apps/foundation-app/src/app/app-routing.module.ts

```diff
@@ -1,8 +1,21 @@
 import { Routes, RouterModule } from '@angular/router';
 import { NgModule } from '@angular/core';
+import { ProductListViewComponent } from './product/pages/product-list-view/product-list-view.component';
+import { CartViewComponent } from './cart/pages/cart-view/cart-view.component';
 
 export const appRoutes: Routes = [
-
+  {
+    path: '',
+    component: ProductListViewComponent
+  },
+  {
+    path: 'product-list',
```

Comment:
This is fine for now, but eventually we'll need to add "category" support. So, there won't be just "product-list", there'll be a wildcard like structure /collection/{whatever}